packages/pharos/src/components/toast/pharos-toaster.ts

```diff
@@ -27,6 +28,13 @@ import { v4 as uuidv4 } from 'uuid';
  * @listens pharos-toast-open
  */
 export class PharosToaster extends PharosElement {
+  /**
+   * The elements to return focus to after closing the toast
+   * @attr returnElements
+   */
+  @property({ type: Array, reflect: true })
```

Comment:
I don't think we want to reflect this as an attribute in the DOM—it will end up rendering something like `returnelements="[Object, Object, ...]"` which you can sort of see happening in the DOM diff at the bottom of [this page](https://www.chromatic.com/test?appId=60919c26122bd50039b34644&id=641c72d990fd1419d4831bd3).

Reply:
updated!